crypto_com_developer_platform_client/integrations/token_api.py

```diff
@@ -4,16 +4,16 @@
 from .api_interfaces import ApiResponse
 
 
-def get_native_token_balance(api_key: str, address: str) -> ApiResponse:
+def get_native_token_balance(api_key: str, wallet_address: str) -> ApiResponse:
     """
     Get the native token balance for a given address.
 
     :param api_key: The API key for authentication.
-    :param address: The address to check the balance for (CronosIds with the `.cro` suffix are supported, e.g. `xyz.cro`)
+    :param wallet_address: The address to check the balance for (CronosIds with the `.cro` suffix are supported, e.g. `xyz.cro`)
     :return: The native token balance.
     :rtype: ApiResponse
     """
-    url = f"{API_URL}/token/native-token-balance?address={address}"
+    url = f"{API_URL}/token/native-token-balance?walletAddress={wallet_address}"
```

Comment:
add wallet_address validation

Reply:
check for any input from user,
such as url injection : containing &,=,% ?

Reply:
example
from urllib.parse import quote
url = f"{API_URL}/token/native-token-balance?walletAddress={quote(wallet_address)}"